app/components/cases/new-subcase.js

```diff
@@ -116,58 +113,69 @@ export default class CasesNewSubcase extends Component {
       created: date,
       modified: date,
       isArchived: false,
+      subcaseName: this.subcaseName,
       agendaActivities: [],
     });
-    subcase.subcaseName = this.subcaseName;
 
+    let piecesFromSubmissions;
     if (this.latestSubcase) {
       // Previous "versions" of this subcase exist
-      subcase = await this.copySubcaseProperties(
+      piecesFromSubmissions = await this.loadSubcasePieces(this.latestSubcase);
+      await this.copySubcaseProperties(
         subcase,
         this.latestSubcase,
-        fullCopy
+        fullCopy,
+        piecesFromSubmissions
       );
     }
-    return subcase;
+    // We save here in order to set the belongsTo relation between submission-activity and subcase
+    await subcase.save();
+
+    if (this.latestSubcase && fullCopy) {
```

Comment:
Dit lijkt de enige echte functionele wijziging (afgezien van de `serialize` optie op het model). Zou meer tijd nodig hebben om dit in-depth te begrijpen, maar twijfel wel of de specs vroeger ms niet gewoon anders begrepen waren. Gaat het hier om een codepad voor élke nieuwe procedurestap (waar er als een eerste bestaat), of enkel de gevallen van "gekloonde" uitgestelde procedurestappen? Die laatste waren (en zijn eigenlijk nog steeds) vanuit een model-oogpunt eerder een tijdelijke oplossing en daarom (met oog op refactor in nabije toekomst) bij de originele implementatie hier misschien niet beschouwd.

Reply:
Die `fullCopy` is enkel `true` in het geval dat er op de knop "kopieer voorgaande procedurestap" wordt gebruikt.
Maar aangezien die knop er ook is bij nieuwe stappen (buiten scope om dat weg te doen), check ik voor de zekerheid nog eens die `this.latestSubcase`